src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkedComponents/TextMeshPro/TextMeshProObserverBase.cs

```diff
@@ -119,8 +121,19 @@ public override void Read(SocketEndpoint sendingEndpoint, BinaryReader message)
                 TextMeshObserver.verticalMapping = (TextureMappingOptions)message.ReadByte();
                 TextMeshObserver.wordWrappingRatios = message.ReadSingle();
                 TextMeshObserver.wordSpacing = message.ReadSingle();
+
+                needsUpdate = true;
             }
 #endif
         }
+
+        protected virtual void Update()
+        {
+            if (needsUpdate)
+            {
+                TextMeshObserver.ForceMeshUpdate();

```

Comment:
>TextMeshObserver.ForceMeshUpdate(); [](start = 16, length = 35)

I think it would be good to put a comment explaining why this is necessary

Reply:
sounds good